ovirtlago/prefix.py

```diff
@@ -49,6 +50,43 @@
 log_task = functools.partial(log_utils.log_task, logger=LOGGER)
 
 
+def _create_junitxml_fullfilename(
+    default_dir, default_filename, junitxml_file=None
+):
+    """
+        Create the file name for the junitxml output
+        default_dir - the default directory for keeping log file
+        default_filename - the default log file name
+
+        Output: results_path ( full path - dir + basename )
+    """
+    if junitxml_file:
+        dirname, basename = os.path.split(junitxml_file)
+        if dirname:
+            default_dir = dirname
+        if basename:
+            default_filename = basename
+
+    results_path = os.path.abspath(
+        os.path.join(
+            default_dir,
+            default_filename,
+        )
+    )
+    return results_path
+
+
+def _safe_makedir(path):
+    """
+        Create a directory recursive
```

Comment:
And don't fail if it already exists.
The short description should be with the quotes in the same line.
```""" Recursively create a directory and don't fail if it's already exist"""```